rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -297,13 +298,15 @@ components:
             The type of product the remote system is. This will help Ion determine how to
             communicate and what functionalities are available.
           minLength: 1
+          maxLength: 20
           example: ddf, alliance, ...
         version:
           type: string
           description: >
             The version of the product installed on the remote system. This will help Ion
             determine how to best communicate with the system.
           minLength: 1
+          maxLength: 80
```

Comment:
❓ Should we put min length of 3 here like the version number above?

Reply:
Since this is provided by the product and I cannot control how to versioned themselves, then `1` to ensure it is not blank is good. For the other versions that are specified by our protocol then `3` was the smallest we kind of support (e.g. `1.2`)